src/lib.rs

```diff
@@ -6,3 +33,230 @@ pub mod providers;
 pub mod relay;
 pub mod state;
 pub mod stores;
+
+pub async fn bootstap() -> error::Result<()> {
```

Comment:
This should take a `config: Config` param, no? That way you don't need any env vars when you bootstrap for tests?